android/engine/src/test/java/org/smartregister/fhircore/engine/util/extension/PatientExtensionTest.kt

```diff
@@ -90,6 +87,13 @@ class PatientExtensionTest : RobolectricTest() {
     Assert.assertEquals(expectedAge14, calculateAge(getDateFromDaysAgo(1826), context))
   }
 
+  @Ignore("Test is flaky and period dependent - it(+ Code) should be refactored")
```

Comment:
Could we add a ticket to track this as _Tech Debt_?

Reply:
Actually it is related to this ticket. You can add it here https://github.com/opensrp/fhircore/issues/2184

Reply:
Ticket created [here](https://github.com/opensrp/fhircore/issues/2260)

Reply:
closed the new ticket and linked it to the earlier one